service/worker/service.go

```diff
@@ -133,7 +133,7 @@ func NewConfig(params *service.BootstrapParams) *Config {
 			Persistence:            &params.PersistenceConfig,
 			ClusterMetadata:        params.ClusterMetadata,
 			TaskListScannerEnabled: dc.GetBoolProperty(dynamicconfig.TaskListScannerEnabled, true),
-			HistoryScannerEnabled:  dc.GetBoolProperty(dynamicconfig.HistoryScannerEnabled, true),
+			HistoryScannerEnabled:  dc.GetBoolProperty(dynamicconfig.HistoryScannerEnabled, false),
```

Comment:
Why do we need to disable this one by default? Concrete execution history has already been disabled I think.

Reply:
I think team agreed to disable scanner by default.

Reply:
I think team agreed to stop execution scanner by default? This one is history scanner for deleting invalid first workflow history batch?